### DISCUSSION TITLE
Can I use this emulate a gamepad

### DISCUSSION THREAD

**Discussion by ShandoTheKing** [AntiMicroX/antimicrox#214] [Q&A] [answered]
I want to emulate a gamepad with my keybard can this be used for that purpose?

**Comment by pktiuk** [comment, marked as answer]
No, AntiMicroX works the opposite way.

It emulates keyboard when you have gamepad

**Reply by pktiuk** [reply]
But it seems to be doable with app called MoltenGamepad (related issue https://github.com/jgeumlek/MoltenGamepad/issues/102)

https://github.com/jgeumlek/MoltenGamepad

**Comment by pktiuk** [comment]
But you can use generated controller mapping SDL2 string in apps like lutris to "remap" our custom gamepad to xbox 360 gamepad.

![obraz](https://user-images.githubusercontent.com/45544416/129900561-8e3703f0-78da-44cc-9d4a-1a77106ea09b.png)

![obraz](https://user-images.githubusercontent.com/45544416/129900690-fce24a8b-94cf-4907-a770-b538bae31380.png)
